lib/chingu/online_high_score_list.rb

```diff
@@ -30,56 +30,62 @@ module Chingu
   #
   class OnlineHighScoreList
     attr_reader :resource, :high_scores
-    
+
     def initialize(options = {})
       @limit = options[:limit] || 100
       @sort_on = options[:sort_on] || :score
       @login = options[:login] || options[:user]
       @password = options[:password]
       @game_id = options[:game_id]
-      
+
       begin
-        require 'rest_client'
-        require 'crack/xml'
+        require "rest_client"
+        require "crack/xml"
       rescue
-        puts "HighScoreList requires 2 gems, please install with:"
+        puts "HighScoreList requires 2 gems, please install them with:"
         puts "gem install rest-client"
         puts "gem install crack"
+
         exit
       end
-      
-      @resource = RestClient::Resource.new("http://api.gamercv.com/games/#{@game_id}/high_scores", 
-                                              :user => @login, :password => @password, :timeout => 20, :open_timeout => 5)
-                                              
-      @high_scores = Array.new  # Keeping a local copy in a ruby array
+
+      @resource = RestClient::Resource.new("http://api.gamercv.com/games/#{@game_id}/high_scores",
```

Comment:
Transferring a password over plain http:// is a red flag, but it looks like this service is defunct anyway. Better to stub it out and leave a comment, or return a single highscore entry indicating that this stopped working.